Restructure (#542)
hacs/documentation: source/docs/use/manage/type/integration.md

title: Integration
---

> Integrations in HACS are community-built integrations.

Author: ```suggestion
Integrations in HACS are community-built custom integrations.

```

## Configuring integrations

> 1. Follow the steps on [downloading a repository](/docs/use/manage/dashboard.md).

Author: ```suggestion
1. Follow the steps on [downloading a repository](/docs/use/download_repository.md).

```

2. How you configure the integration in Home Assistant depends on the integration:

    - The integration documentation page in HACS should instruct you how to get started.

>     - Some integrations, you can add from the integration page on <!-- hacs:my integrations title="**{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}**" -->.

Author: ```suggestion

    - Some integrations can be added from the integration page on <!-- hacs:my integrations title="**{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}**" -->.
    - Others need to be added via YAML.
3. **Troubleshooting**: If you select the **Add integration** button but the integration does not show, clear your browser cache to refresh the UI.
```

Integrations in HACS are community-built custom integrations.

## Configuring integrations

1. Follow the steps on [downloading a repository](/docs/use/download_repository.md).

2. How you configure the integration in Home Assistant depends on the integration:

    - The integration documentation page in HACS should instruct you how to get started.

    - Some integrations can be added from the integration page on <!-- hacs:my integrations title="**{{coreui('panel.config')}}** > **{{coreui('ui.panel.config.dashboard.devices.main')}}**" -->.
    - Others need to be added via YAML.
3. **Troubleshooting**: If you select the **Add integration** button but the integration does not show, clear your browser cache to refresh the UI.

>     - Others, you need to add via YAML.
>     - If you select the **Add integration** button and the integration does not show, clear your browser cache to refresh the UI.

Author: ```suggestion
```

Integrations in HACS are community-built custom integrations.

> 

Author: ```suggestion
```

Integrations in HACS are community-built custom integrations.

## Configuring integrations

1. Follow the steps on [downloading a repository](/docs/use/download_repository.md).

> 

Author: ```suggestion
```
